src/dialogs/shortcuts/dialog-shortcuts.ts

```diff
@@ -161,7 +162,18 @@ class DialogShortcuts extends LitElement {
 
     return html`
       <div class="shortcut">
-        ${keys.map((key) => html` <span>${key.toUpperCase()}</span>`)}
+        ${keys.map(
+          (key) =>
+            html` <span
+              >${key === CTRL_CMD
+                ? isMac
+                  ? html`<ha-svg-icon
+                      .path=${mdiAppleKeyboardCommand}
+                    ></ha-svg-icon>`
+                  : this.hass.localize("ui.panel.config.automation.editor.ctrl")
+                : key.toUpperCase()}</span
```

Comment:
```suggestion
                : key}</span
```

it is already as uppercase defined... If we want to change it to lower case we could do it then easily.